internal/commands/scan_test.go

```diff
@@ -1187,3 +1192,40 @@ func TestValidateContainerImageFormat(t *testing.T) {
 		})
 	}
 }
+
+func TestRunScaResolverAndAddScaResultsFileCleanup(t *testing.T) {
```

Comment:
we already have logic for reading logs from file
look for "createOutputFile" method, and see it's uses